agent-discoveries-frontend/src/main.tsx

```diff
@@ -2,9 +2,10 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App'
 import './index.css'
+import ProfilePage from './Views/ProfilePage/ProfilePage'
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
-    <App />
+    <ProfilePage />
```

Comment:
You're going to want to revert this and co-ordinate with Jamie about how the routing's going to work